scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/cipher/CipherSuite.java

```diff
@@ -372,6 +372,25 @@ public static int getOverallMaxCiphertextExpansion() {
 		return overallMaxCipherTextExpansion;
 	}
 
+	/**
+	 * Get list of supported cipher suites.
+	 * 
+	 * Some cipher suites are not supported by all JVM (especially java 1.7).
+	 * 
+	 * @param cipherSuites list of cipher suites to be filtered based on
+	 *            {@link #isSupported()}.
+	 * @return list of supported cipher suites
+	 */
+	public static List<CipherSuite> getSupportedCipherSuites(List<CipherSuite> cipherSuites) {
+		List<CipherSuite> list = new ArrayList<>();
+		for (CipherSuite suite : cipherSuites) {
+			if (suite.isSupported()) {
+				list.add(suite);
+			}
+		}
+		return list;
+	}
+
```

Comment:
Just in case, it is not intended : Eclipse says me that this method in not currently used.
(But I'm ok to add to the API)

Reply:
I removed it.